requirements.txt

```
@@ -4,4 +4,4 @@ pymdown-extensions>=7.0.0
 mkdocs-branchcustomization-plugin~=0.1.3
 mkdocs-macros-plugin
 numpy
-tools/schemacode/[render]
+-e tools/schemacode/[render]
```

I think I suggested this, but I disagree now. This is part of the build process in RTD, and we shouldn't rely on `-e`.

```suggestion
tools/schemacode/[render]
```